lib/tasks/load_blacklist.rake

```diff
@@ -9,4 +10,8 @@ task :load_blacklist, [:blacklist_file] => :environment do |_t, args|
       tl.save
     end
   end
+  # update the pipeline runs
+  pipeline_runs = PipelineRun.where("id in (select max(id) from pipeline_runs where job_status = 'CHECKED' and sample_id in (select id from samples) group by sample_id)")
```

Comment:
should we make this a function? We've repeated the same query a few times

Reply:
yeah. great idea.